mmdet/integration/nncf/compression.py

```diff
@@ -108,11 +113,36 @@ def get_inputs(self, dataloader_output):
 
     pathlib.Path(cfg.work_dir).mkdir(parents=True, exist_ok=True)
     nncf_config = NNCFConfig(cfg.nncf_config)
+    nncf_config.target_metric_name = cfg.nncf_config.target_metric_name
```

Comment:
Running the compression without accuracy_aware_training in config causes `AttributeError: 'ConfigDict' object has no attribute 'target_metric_name'`

Reply:
Fixed